app/src/main/java/com/cmput301f20t21/bookfriends/ui/library/BorrowedListFragment.java

```diff
@@ -41,8 +45,16 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         recyclerView.setLayoutManager(layoutManager);
 
         // specify an adapter (see also next example)
-        mAdapter = new BorrowedListAdapter(mViewModel.getBooks());
-        recyclerView.setAdapter(mAdapter);
+        vm.getBooks().observe(getViewLifecycleOwner(), (List<Book> books) -> {
+            adapter = new BorrowedListAdapter(books);
```

Comment:
one little thing there if you don't specify an adapter directly in the onViewCreated() it will thrown an error
![image](https://user-images.githubusercontent.com/43459469/97526398-a200f200-196e-11eb-9e32-4bf9815cc19c.png)

Not sure if this error will affect anything, but a simple way to get rid of the error is to feed in a temporary adapter, once we get the data we will set it to the real adapter